packages/office-ui-fabric-react/src/components/Tooltip/Tooltip.base.tsx

```diff
@@ -47,24 +47,26 @@ export class TooltipBase extends React.Component<ITooltipProps, any> {
     });
 
     return (
-      <Callout
-        target={targetElement}
-        directionalHint={directionalHint}
-        directionalHintForRTL={directionalHintForRTL}
-        {...calloutProps}
-        {...getNativeProps(this.props, divProperties, ['id'])} // omitting ID due to it being used in the div below
-        className={this._classNames.root}
-      >
-        <div
-          className={this._classNames.content}
-          id={id}
-          role="tooltip"
-          onMouseEnter={this.props.onMouseEnter}
-          onMouseLeave={this.props.onMouseLeave}
+      <DelayedRender delay={delay}>
```

Comment:
delay value is an enum. you'll need to do a quick case switch for the 3 matches. 0, 300, 500ms

Reply:
Also (as mentioned) remove it from the styles file

Reply:
This needs to get addressed before your tests pass.

Reply:
I thought I addressed this with the following: https://github.com/OfficeDev/office-ui-fabric-react/pull/9341/files#diff-00719f3cbd47c1407b8d401275c4292eR48 and I did remove it from the styles file altogether: https://github.com/OfficeDev/office-ui-fabric-react/pull/9341/files#diff-797904568f02429664f4dc0012b330b9L16